ed448-goldilocks/src/constants.rs

```diff
@@ -12,9 +12,3 @@ pub const DECAF_BASEPOINT: DecafPoint = DecafPoint(curve::twedwards::extended::E
 /// \ell = 2^\{446\} + 0x8335dc163bb124b65129c96fde933d8d723a70aadc873d6d54a7bb0d.
 /// $$
 pub const EDWARDS_BASEPOINT_ORDER: EdwardsScalar = EdwardsScalar::new(ORDER);
-
-/// `BASEPOINT_ORDER` is the order of the Decaf448 basepoint, i.e.,
-/// $$
-/// \ell = 2^\{446\} + 0x8335dc163bb124b65129c96fde933d8d723a70aadc873d6d54a7bb0d.
-/// $$
-pub const DECAF_BASEPOINT_ORDER: DecafScalar = DecafScalar::new(ORDER);
```

Comment:
Hmm, these both seem problematic, especially in the `pub` API, as they're out-of-range scalars (i.e. the highest valid scalar is `ORDER - 1`)